extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qui/qui-badge.js

```diff
@@ -80,6 +80,7 @@ export class QuiBadge extends LitElement {
         icon: {type: String},
         level: {type: String},
         small: {type: Boolean},
+        warning: {type: Boolean},
```

Comment:
You do not need to add this. You can just use it as is: see this for a warning example: https://github.com/phillip-kruger/quarkus-jokes/blob/63db547df6f0b548cb1cebf47062da23ef4b7b95/deployment/src/main/resources/dev-ui/qwc-jokes-vaadin.js#LL222C29-L222C29

You can remove all your modification in qui-badge

Reply:
@phillip-kruger  ok ! If you agree I will delete only line 98 where "this.small = false;" is duplicated.